app-modules/service-management/tests/Filament/ServiceMonitoring/Pages/CreateServiceMonitoringTest.php

```diff
@@ -139,3 +135,45 @@
     expect($serviceMonitoringTarget->teams())->exists()->toBeTrue();
     expect($serviceMonitoringTarget->users())->exists()->toBeTrue();
 });
+
+test('it will validate multiple valid forms of URL and IP Address', function (array $validUrls, array $invalidUrls) {
+    asSuperAdmin();
+
+    foreach ($validUrls as $url) {
+        $request = ServiceMonitoringTarget::factory()->make(['domain' => $url])->toArray();
+
+        livewire(CreateServiceMonitoring::class)
+            ->fillForm($request)
+            ->call('create')
+            ->assertHasNoFormErrors();
+    }
+
+    foreach ($invalidUrls as $url) {
+        $request = ServiceMonitoringTarget::factory()->make(['domain' => $url])->toArray();
+
+        livewire(CreateServiceMonitoring::class)
+            ->fillForm($request)
+            ->call('create')
+            ->assertHasFormErrors(['domain']);
+    }
+})
+    ->with([
+        [
+            'validUrls' => [
+                'http://example.com',
+                'https://test.com',
+                'http://192.168.0.1',
+                'http://[2001:db8::1]',
+                'https://[fe80::1ff:fe23:4567:890a]:443',
```

Comment:
Hmm, I had mentioned in one of my last comments that an IP address typically is not provided with protocol.

Provide IP addresses without protocol and they should be valid.